code/modules/mob/living/basic/pets/parrot/parrot_ai/parrot_controller.dm

```diff
@@ -0,0 +1,251 @@
+/datum/ai_controller/basic_controller/parrot
```

Comment:
this should be split up into more files, e.g. "parrot_hoarding.dm" for all the hoarding stuff, etc.- just too long right now